README.md

```diff
@@ -1,364 +1,459 @@
 WORKDIR /app
 EXPOSE 5000
 
-# copy csproj and restore as distinct layers
+# Copy csproj and restore as distinct layers
 COPY *.csproj ./
 RUN dotnet restore
 
-# copy everything else and build
+# Copy everything else and build
 COPY . ./
 RUN dotnet publish -c Release -o out
-<b>
-# build runtime image
-FROM microsoft/dotnet:2.1.0-aspnetcore-runtime
+
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
 WORKDIR /app
 COPY --from=build-env /app/out .
 ENTRYPOINT ["dotnet", "TodoApi.dll"]
-</b>
-</code></pre>
-> **IMPORTANT** You want to make sure that the dll referenced in the last line of your dockerfile (TodoApi.dll in this case) matches what will be produced by your application.  Meaning if you named your application something other than TodoApi then you will need to change this line.  If you dont, you will get a strange error like the following <b>Did you mean to run dotnet SDK commands? Please install dotnet SDK from</b> and you wont know what is causing it. 
-
-
+```
 
+> **IMPORTANT** You want to make sure that the dll referenced in the last line of your Dockerfile (TodoApi.dll in this
+> case) matches what will be produced by your application. Meaning if you named your application something other than
+> TodoApi then you will need to change this line. If you don't, you will get a strange error like, "Did you mean to run
+> dotnet SDK commands? Please install dotnet SDK," and you won't know what is causing it.
 
+We also want to create a **.dockerignore** (don't forget the '.' in front of the name) this will make sure we keep the
+image as fast and as small as possible by ignoring files we don't care about. Place it in the root of your project
+directory (same as Dockerfile). We are excluding the bin and obj folders, as well as stuff associated with VS Code and
+Git, but you can add anything you don't need packaged in the container.
 
-We also want to create a <b>.dockerignore</b>  (don't forget the '.' in front of the name) this will make sure we keep the image as fast and as small as possible by ignoring files we don't care about. Place it in the root of your project directory (same as Dockerfile). We are excluding the bin and obj folders, as well as stuff associated with VSCode and git,  but you can add anything you don't need packaged in the container. 
-<pre><code><b>.dockerignore
+```dockerignore
+.dockerignore
 .env
 .git
 .gitignore
 .vs
 .vscode
 */bin
-*/obj</b>
-</code></pre>
---
-
-Now we want to create our images using docker at the command line.  Open up your terminal (Command Window, PowerShell, Terminal), CD (Change Directory) into the folder that holds your project (and Dockerfile),  and type the following commands
-
-Don't forget the '.' at the end. This will build our image according to our Dockerfile and give it a name of todov1.  It will be tagged as todov1:latest.  If we want our own tag we just add it in the command todov1:testversion or todov1:version1.0
-
-```
--> docker build -t todov1 . 
```

Comment:
Removed all the `-->` that were prefixed to commands so that users can easily copy and paste into their terminal with GitHub's clipboard button